src/note-db.ts

```diff
@@ -105,10 +105,20 @@ const addNoteToDb = (note: Note) => {
     vscode.window.showInformationMessage('Annotation saved!');
 };
 
+const getTODOFromSelectedText = (): string | undefined => {
+    const editor = vscode.window.activeTextEditor;
+    const selectedText = editor?.selection ? editor.document.getText(editor.selection) : '';
+    const todoSelector = /\/\/ TODO: (.*)\n/;
+    let matchArray = selectedText.match(todoSelector);
+    return matchArray?.length ? matchArray[1] : undefined;
+};
+
 export const addNote = async () => {
     const editor = vscode.window.activeTextEditor;
     if (editor) {
-        const annotationText = await vscode.window.showInputBox({ placeHolder: 'Give the annotation some text...' });
+        const todoText = getTODOFromSelectedText();
+        let annotationText = await vscode.window.showInputBox({ placeHolder: todoText ?? 'Give the annotation some text...' });
```

Comment:
I was testing here, and I think it would be better to display this as a text. I believe something like the following should work:

```suggestion
        let annotationText = await vscode.window.showInputBox({ placeHolder: 'Give the annotation some text...', value: todoText });
```

Reply:
The difference is that with my suggestion the text would be there itself. Hitting enter would save that as is, but clearing it should also be simple enough.